spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'Takeaway'
+
+describe Takeaway do
+
+  subject(:takeaway1) {described_class.new}
+  order_hash = {"chicken noodle soup" => 2,
+           "burger and fries" => 1,
+           "chicken tikka madras" => 4,
+           "fresh caesar salad" => 1}
+
+  wrong_order = {"chicken noodle soup" => 1,"chicken tikka madras" => 1, 'beans on toast' => 1}
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)